server/src/controllers/user.controller.js

```diff
@@ -14,7 +12,8 @@ export async function checkRole(req, res) {
     }
   }
   else {
-    res.json({ role: "user" })
+    // add new non-admin user to DB?
```

Comment:
I don't personally see a need to store users in the db while it's assumed as a default (ie any random person can come and immediately request an appointment)

Reply:
Makes sense to me as well! Wasn't sure if that was something we should do. Thanks 👍